toolchain/check/handle_interface.cpp

```diff
@@ -63,6 +63,8 @@ static auto BuildInterfaceDecl(Context& context,
     if (auto existing_interface_decl =
             context.insts().Get(existing_id).TryAs<SemIR::InterfaceDecl>()) {
       // This is a redeclaration of an existing interface.
+      // TODO: class_decl = *existing_class_decl;
+      interface_decl.type_id = existing_interface_decl->type_id;
```

Comment:
I don't understand this change: don't we always want the type to be `TypeType`? I would think the only thing we need to update here is the `interface_id`.

Reply:
Sorry, missed cleaning this up (leftover from while editing)